docs-requirements.txt

```diff
@@ -1,33 +1,33 @@
 -i https://pypi.python.org/simple
-alabaster==0.7.13 ; python_version >= '3.6'
-babel==2.12.1 ; python_version >= '3.7'
-certifi==2023.7.22 ; python_version >= '3.6'
-charset-normalizer==2.1.1
-click==8.1.7 ; python_version >= '3.7'
-click-default-group==1.2.4 ; python_version >= '2.7'
-docutils==0.17.1
-idna==3.4 ; python_version >= '3.5'
-imagesize==1.4.1 ; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
-importlib-resources==6.0.1 ; python_version < '3.10'
+alabaster==0.7.13; python_version >= '3.6'
+babel==2.12.1; python_version >= '3.7'
+certifi==2023.7.22; python_version >= '3.6'
+charset-normalizer==2.1.1; python_full_version >= '3.6.0'
```

Comment:
Just a question: IIRC, `nucypher` supports Python 3.8+. The contents or `requirements.txt` files etc., are supposed to reflect the minimum supporter Python version, right?

Reply:
I think so.

In any case, requirements.txt is generated by `relock_dependencies` script. Not sure what is the logic behind all that `python_version` tags. Maybe is the `python_version` that all those packages need?

Reply:
These are autogenerated by pip when relocking and doesn't take into account `nucypher`'s specific minimum supported version. Some of the restrictions are applicable while others are redundant. For example, https://github.com/nucypher/nucypher/blob/development/requirements.txt#L9, since we support 3.8, 3.9, 3.10. That dependency only applies to < 3.9.